### PR TITLE
Audit sites for our short URLs that use go.usa.gov

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary

Search and replace shortened urls to their absolute url because `go.usa.gov` will be sunset **9/15**.

## Preview

[Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-update-shortened-urls/)

## Problem

`go.usa.gov` service is set for sunsetting **9/15** and shortened links need to be replaced with the absolute url.

## Solution

Search and replace for the following:

  - replaces shortened url with absolute url in content markup
  - removes `short_url` from front matter
  - keep content links to `go.usa.gov` in existing posts

Remove `get_clicks.html` partial from templates that display metrics counter.

- layouts/_default/baseof.html
- layouts/events/card-event-past.html
- layouts/news/card-article.html
- layouts/news/card-elsewhere.html
- layouts/news/single.html

### CODE REVIEW

**Review by mejiaj — CHANGES_REQUESTED** [review]
Overall looks pretty good. There are some files that have extra lines and added some comments on other findings.

File: content/news/2013/12/2013-12-20-gov-url-shorteners-and-how-to-use-them.md [hunk 1]
```diff
@@ -58,4 +58,4 @@ URL shorteners that use 301 “permanent” redirects do not have negative effec
 
 Commercial search engines treat short URLs with 301 redirects just like any other link. The short URL’s destination page will receive the social promotion bump and other SEO data (such as anchor text and position on the page) will be passed on too.
 
-This video explains <a href="http://www.youtube.com/watch?v=QMkltd6dZzU">how Google treats short URLs</a>. For more on 301 redirects, read these tips from <a href="http://www.bing.com/blogs/site_blogs/b/webmaster/archive/2011/10/06/managing-redirects-301s-302s-and-canonicals.aspx">Bing</a> and from <a href="https://support.google.com/webmasters/answer/93633">Google</a>.
+This video explains <a href="http://www.youtube.com/watch?v=QMkltd6dZzU">how Google treats short URLs</a>. For more on 301 redirects, read these tips from <a href="http://www.bing.com/blogs/site_blogs/b/webmaster/archive/2011/10/06/managing-redirects-301s-302s-and-canonicals.aspx">Bing</a> and from <a href="https://support.google.com/webmasters/answer/93633">Google</a>.
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/news/2013/12/2013-12-20-gov-url-shorteners-and-how-to-use-them.md:61]: I've updated the link to the new working URL

File: content/events/2020/06/2020-06-30-10x-project-lightning-talks-2020.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Through a series of quick presentations, we’ll learn more about the following
  - Mike Stern, Government Notification Services
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/events/2020/06/2020-06-30-10x-project-lightning-talks-2020.md:77]: Good catch, this is now fixed

File: content/news/2020/04/2020-04-24-experimenting-with-micropurchase-threshold.md [hunk 3]
```diff
@@ -3,8 +3,7 @@
 # Learn how to edit our pages at https://workflow.digital.gov
 slug: experimenting-with-micropurchase-threshold
 
-# Short URL — https://go.usa.gov/
-short_url: https://go.usa.gov/xvQ7a
+
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/news/2020/04/2020-04-24-experimenting-with-micropurchase-threshold.md:6]: Extra space has been removed

File: content/news/2020/05/2020-05-20-tapping-into-seo-how-government-websites.md [hunk 4]
```diff
@@ -3,8 +3,7 @@
 # Learn how to edit our pages at https://workflow.digital.gov
 slug: tapping-into-seo-how-government-websites
 
-# Short URL — https://go.usa.gov/
-short_url: https://go.usa.gov/xvhmA
+
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/news/2020/05/2020-05-20-tapping-into-seo-how-government-websites.md:6]: This is now removed

File: content/news/2020/05/2020-05-27-federal-source-code-study-series.md [hunk 5]
```diff
@@ -10,8 +10,7 @@ source_url: "https://medium.com/codedotgov/federal-source-code-study-series-part
 source: codegov
 slug: federal-source-code-study-series
 
-# Short URL — https://go.usa.gov/
-short_url: 
+
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/news/2020/05/2020-05-27-federal-source-code-study-series.md:13]: This is now removed

File: themes/digital.gov/src/js/common/clicks.js [hunk 6]
```diff
@@ -1,109 +0,0 @@
...
-        // Count up from the existing number to the current_num_clicks
-        var interval = setInterval(function() {
-          if (stale_clicks >= current_clicks) clearInterval(interval);
-          stale_clicks++;
-          $(element_span).html(stale_clicks);
-        }, 50);
-      });
-    });
-  });
-});
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/src/js/common/clicks.js:109]: Removed `_clicks.scss` as well as removed the comment section in `baseof.html`. Yes, we will be getting rid of this tracking info for the usa.go.gov sunsetting.

File: content/news/2015/09/2015-09-28-nasa-lunar-eclipse-a-supermoon-match-made-in-metrics-heaven.md [hunk 7]
```diff
@@ -1,6 +1,5 @@
 ---
 slug: nasa-lunar-eclipse-a-supermoon-match-made-in-metrics-heaven
-short_url:
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply content/news/2015/09/2015-09-28-nasa-lunar-eclipse-a-supermoon-match-made-in-metrics-heaven.md:3]: I have removed this, @ToniBonittoGSA should it stay?

File: themes/digital.gov/src/scss/new/styles.scss [hunk 8]
```diff
@@ -162,7 +162,6 @@
 @forward 'list-topics';
 
 @forward 'breadcrumbs';
-@forward 'clicks';
```
Comment (mejiaj) [inline themes/digital.gov/src/scss/new/styles.scss:165]: We can remove click styles on cards too

https://github.com/GSA/digitalgov.gov/blob/23a07f637260ca0c1167bc83bddaa06eddba67ff/themes/digital.gov/src/scss/new/_card.scss#L75-L89

**Review by mejiaj — CHANGES_REQUESTED** [review]
Minor note on some additional click styles, otherwise LGTM!

File: themes/digital.gov/src/scss/new/styles.scss (hunk 8 above)
Comment (mejiaj) [reply themes/digital.gov/src/scss/new/styles.scss:165]: Removed in [65e10a8](https://github.com/GSA/digitalgov.gov/pull/5310/commits/65e10a80365b58d02d65fce754210bc71b46f4d0)

File: themes/digital.gov/src/js/common/clicks.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (mejiaj) [reply themes/digital.gov/src/js/common/clicks.js:109]: Removed in [23a07f6](https://github.com/GSA/digitalgov.gov/pull/5310/commits/23a07f637260ca0c1167bc83bddaa06eddba67ff)

**Review by mejiaj — APPROVED** [review]
LGTM, thanks Nick!

**Pull request merged**